### PR TITLE
corrected README for location of Jupyter chart examples

### PR DESCRIPTION
**Pull request by gregorydillon**
Issue created at #5638

trivial adjustment to README.md file

### CODE REVIEW

File: examples/charts/README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ There are two subdirectories:
 
 * `file` examples that output to static HTML files
 
+* `server`
+
+The example code for charts using Jupyter/Ipython notebooks is located under "howto"
+../howto/charts
```
Comment (bryevdv) [inline examples/charts/README.md]: I think if you use the markdown syntax, i.e. `[text](address)` this will render as a nice link

**Comment by bryevdv** [comment]
@gregorydillonI think the old text regarding `charts/notebook` should be removed as well

**Comment by bryevdv** [comment]
Hi @gregorydillon just checking in, did you intend to continue working on this PR?

**Comment by gregorydillon** [comment]
Yes thanks, New Year distraction.

Will do today

**Comment by bryevdv** [comment]
Great thanks! No worries was just checking in for planning purposes :)

**Comment by bryevdv** [comment]
@gregorydillon checking in again. I understand if you do not have more time to work on this, but please let me know if that is the case so that I may close it.

**Comment by bryevdv** [comment]
@gregorydillon I have gone ahead and made the other changes, thanks for the PR

**Pull request merged**